### DISCUSSION TITLE
Jenkins Integration with EJBCA 8.0 .

### DISCUSSION THREAD

**Discussion by ajitdu** [Keyfactor/ejbca-ce#588] [Q&A]
Use Case: Integrate Jenkins with EJBCA for Certificate Generation.

Plan: Request comes to Jira for a certificate, once Jira is approved a Jenkins Job is triggered to make a request to EJBCA to generate a certificate using a CSR.

Has any one does this integration before? Please help with your suggestions and ideas.

Thanks

**Comment by ajitdu** [comment]
Hi Sven ,

Can you please help with this?

Thanks

**Comment by primetomas** [comment]
You could use something similar to this SignServer integration. You just need to modify the script and use the EJBCA REST API to issue the certificate.
https://doc.primekey.com/signserver/tutorials-and-guides/how-to-integrate-jenkins-with-signserver-for-automated-code-signing
